maven/test/unit/src/org/netbeans/modules/maven/newproject/simplewizard/SimpleWizard.html

```diff
@@ -1,3 +1,21 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
```

Comment:
This XML license header also seems unline the others.

Reply:
Sam as comment above

Reply:
Copy it from an existing file in the netbeans codebase. For example, for maven/

XML file: see maven/build.xml
.properties file: see maven/nbproject/project.properties
.java file: see some existing Java file in maven/src

etc.

Reply:
Ok. Thanks